core/src/main/java/com/hcl/ecomm/core/services/impl/LoginServiceImpl.java

```diff
@@ -0,0 +1,87 @@
+package com.hcl.ecomm.core.services.impl;
+
+import com.hcl.ecomm.core.config.LoginServiceConfig;
+import com.hcl.ecomm.core.services.LoginService;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.propertytypes.ServiceDescription;
+import org.osgi.service.component.propertytypes.ServiceRanking;
+import org.osgi.service.component.propertytypes.ServiceVendor;
+import org.osgi.service.metatype.annotations.Designate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Component(service = { LoginService.class })
+@Designate(ocd = LoginServiceConfig.class)
+
+@ServiceDescription("Login service")
+@ServiceRanking(1001)
+@ServiceVendor("Nithesh")
```

Comment:
@nitheshkumar-hcl this should be hcl

Reply:
Committed the changes as per the review comments